<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myWidget</class>
 <widget class="QWidget" name="myWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>441</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>myWidget</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/background.png);</string>
  </property>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>508</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
font: 75 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>Accesskey</string>
   </property>
  </widget>
  <widget class="QLabel" name="password">
   <property name="geometry">
    <rect>
     <x>508</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
font: 75 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>Secretkey</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logpushButton">
   <property name="geometry">
    <rect>
     <x>709</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
font: 75 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="resetpushButton">
   <property name="geometry">
    <rect>
     <x>598</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
font: 75 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="secretKeylineEdit">
   <property name="geometry">
    <rect>
     <x>598</x>
     <y>120</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="accessKeylineEdit">
   <property name="geometry">
    <rect>
     <x>598</x>
     <y>80</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>211</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-size:10pt; font-weight:600;&quot;&gt;Accesskey和Secretkey的获取方法：&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-size:10pt; font-weight:72;&quot;&gt;登录移动云官网（https://ecloud.10086.cn),搜索“云API&amp;quot;进入控制台界面，点击”创建“，按照提示下载表格，下载的表格内即是Accesskey和Secretkey。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
